src/Microsoft.OData.Client/Microsoft.OData.Client.csproj

```diff
@@ -1,14 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net45;netstandard2.0;</TargetFrameworks>
+    <TargetFrameworks>net8.0;net48</TargetFrameworks>
     <AssemblyName>Microsoft.OData.Client</AssemblyName>
     <RootNamespace>Microsoft.OData.Client</RootNamespace>
     <DocumentationFile>$(AssemblyName).xml</DocumentationFile>
     <DefineConstants>$(DefineConstants);ODATA_CLIENT;SUPPRESS_PORTABLELIB_TARGETFRAMEWORK_ATTRIBUTE;DelaySignKeys</DefineConstants>
    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net8.0'">

```

Comment:
Looking at what is nested within this element, I don't think we need this `PropertyGroup` at all

Reply:
Okay. I was reluctant to remove things I was not sure about. But if you're sure it's not used I can go ahead and remove it.

Reply:
@habbes What is nested is `<DefineConstants>$(DefineConstants);ASSEMBLY_ATTRIBUTE_ON_NETSTANDARD_20</DefineConstants>`. Clearly, this is only relevant to NetStandard 2.0 so it wouldn't have any relevance after we remove support for that standard.

Reply:
Removed.

Reply:
Removing this caused compilation errors. There's some code which depends on this flag. Maybe we can revisit this later?

Reply:
I could create a work item to perform clean up after this so that this PR doesn't block other work from taking place.